include/deal.II/fe/fe_series.h

```diff
@@ -307,7 +359,7 @@ namespace FESeries
     /**
      * hp::QCollection used in calculation of transformation matrices.
      */
-    SmartPointer<const hp::QCollection<dim>> q_collection;
+    const hp::QCollection<dim> q_collection;
```

Comment:
This change causes an error in line 615 for the serialization below. Can you adjust that line @tamiko (I can't click there)?
```c++
size = q_collection.size();
```